pom.xml

```diff
@@ -60,7 +62,107 @@
 						<target>1.7</target>
 					</configuration>
 				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-release-plugin</artifactId>
+					<version>2.5</version>
+					<configuration>
+						<autoVersionSubmodules>true</autoVersionSubmodules>
+						<useReleaseProfile>false</useReleaseProfile>
+						<releaseProfiles>release</releaseProfiles>
+						<goals>deploy</goals>
+						<tagNameFormat>v@{project.version}</tagNameFormat>
+						<arguments>-Prelease</arguments>
+					</configuration>
+				</plugin>
 			</plugins>
 		</pluginManagement>
 	</build>
+
+	<distributionManagement>
+		<repository>
+			<id>sonatype-nexus-staging</id>
+			<name>Nexus Release Repository</name>
+			<url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+		</repository>
+		<snapshotRepository>
+			<id>sonatype-nexus-snapshots</id>
+			<name>Sonatype Nexus Snapshots</name>
+			<url>https://oss.sonatype.org/content/repositories/snapshots/</url>
+		</snapshotRepository>
+	</distributionManagement>
+
+	<scm>
+		<connection>scm:git:git@github.com:onelogin/java-saml.git</connection>
+		<developerConnection>scm:git:git@github.com:onelogin/java-saml.git</developerConnection>
+		<url>https://github.com/onelogin/java-saml</url>
+		<tag>HEAD</tag>
```

Comment:
Maybe reference for a v2.0.0 tag that I can create in the same way I generated the [v.1.1.2](https://github.com/onelogin/java-saml/releases/tag/v1.1.2)?